### PR TITLE
[Task] Update subscribers and docs

### LINKED ISSUE #498: [Task] Adapt Subscribers and docs for firewall security

### PR DESCRIPTION
**Pull request by mattamon**
## Changes in this pull request
Resolves #498 

## Additional info
Updating subscribers and docs to reflect the new prefix

### CODE REVIEW

File: doc/00_Installation.md [hunk 1]
```diff
@@ -110,3 +111,21 @@ pimcore_studio_backend:
...
+Keep in mind that you need to update your access_control settings accordingly.
+```yaml
+pimcore_studio_backend:
+  url_prefix: '/your-prefix/api/'
+```
+
+```yaml
+security:
+    firewalls: 
+        pimcore_studio: '%pimcore_studio_backend.firewall_settings%'
```
Comment (martineiber) [inline doc/00_Installation.md]: Those two lines can be removed? Has nothing ToDo with the prefix change?

**Review by martineiber — CHANGES_REQUESTED** [review]
Rest looks good

File: doc/00_Installation.md (hunk 1 above)
Comment (mattamon) [reply doc/00_Installation.md]: That is true, I just wanted to give the complete picture again. But I can also remove them.

Comment (mattamon) [reply doc/00_Installation.md]: Done :)

**Review by martineiber — APPROVED** [review]
LGTM :duck:

**Pull request merged**